code/style.css

```diff
@@ -17,6 +33,13 @@ p {
   display: flex;
   justify-content: space-around;
   align-items: center;
+  background-image: url(assets/hauntedhouse.jpg);
```

Comment:
`    background-size: cover;`
might be super useful here.  Give it a try and google what it does 👍